app/src/main/res/menu/menu.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/money"
+        android:icon="@drawable/ic_attach_money_black_24dp"
+        android:title="text"
```

Comment:
The title here should be money or something of the sort no? We can change this in another PR.

Reply:
What do you mean by this ?

Reply:
You are right kev. good catch I'll update it

Reply:
I put the proper title. Waiting on CI to go green then merging it